From 9bfcb0be3ff34354c9299bf6e0a90e691f14e813 Mon Sep 17 00:00:00 2001
From: Samuel Verschelde <stormi-xcp@ylix.fr>
Date: Fri, 27 Jan 2023 12:03:15 +0100
Subject: [PATCH 17/23] Tell users not to edit multipath.conf directly

This file is meant to remain unchanged and regularly updated along with
the SM component. Users can create a custom configuration file in
/etc/multipath/conf.d/ instead.

Signed-off-by: Samuel Verschelde <stormi-xcp@ylix.fr>
(cherry picked from commit b44d3f5db6d22d5a649fffebebb284afd6da39f7)
---
 multipath/multipath.conf | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/multipath/multipath.conf b/multipath/multipath.conf
index f63506d8..c854b577 100644
--- a/multipath/multipath.conf
+++ b/multipath/multipath.conf
@@ -1,3 +1,11 @@
+# --- WARNING: DO NOT EDIT THIS FILE ---
+# The contents of this file may be overwritten at any future time through a
+# system update, causing any custom configuration to be lost.
+#
+# For custom multipath configuration, create a separate .conf file in the
+# /etc/multipath/conf.d/ directory.
+# --- END OF WARNING ---
+
 # This configuration file is used to overwrite the built-in configuration of
 # multipathd.
 # For information on the syntax refer to `man multipath.conf` and the examples
